local-build-plugins/src/main/java/org/hibernate/orm/docs/PublishTask.java

```diff
@@ -54,11 +54,12 @@ public void uploadDocumentation() {
 		final String url = normalizedBase + releaseFamily;
 
 		final String stagingDirPath = stagingDirectory.get().getAsFile().getAbsolutePath();
+		final String stagingDirPathContent =  stagingDirPath.endsWith( "/" ) ? stagingDirPath : stagingDirPath + "/";
```

Comment:
It is likely that `stagingDirPath` never ends '/' for internally it will go through a normalizing processing, though it is not a big deal and doing the redundant checking doesn't hurt.